From bbcc4e044af1df2626b321613cdbd931c2037a95 Mon Sep 17 00:00:00 2001
From: LuK1337 <priv.luk@gmail.com>
Date: Sun, 31 Dec 2017 15:25:59 +0100
Subject: [PATCH 17/21] msm8953-common: Remove AOSP seccomp policy

While it served the purpose so far, no longer works when using
CAF media HAL. Remove it to allow clean picks from CAf.

Change-Id: I50edd601c24b40b735dd167435ae4a1d77ea76ad
---
 msm8953.mk                       |  4 ----
 seccomp_policy/mediacodec.policy | 12 ------------
 2 files changed, 16 deletions(-)
 delete mode 100644 seccomp_policy/mediacodec.policy

diff --git a/msm8953.mk b/msm8953.mk
index 3ccc021..f201eb1 100644
--- a/msm8953.mk
+++ b/msm8953.mk
@@ -310,10 +310,6 @@ PRODUCT_PACKAGES += \
 PRODUCT_BOOT_JARS += \
     telephony-ext
 
-# Seccomp policy
-PRODUCT_COPY_FILES += \
-    $(LOCAL_PATH)/seccomp_policy/mediacodec.policy:$(TARGET_COPY_OUT_VENDOR)/etc/seccomp_policy/mediacodec.policy
-
 # Sensors
 PRODUCT_PACKAGES += \
     android.hardware.sensors@1.0-impl \
diff --git a/seccomp_policy/mediacodec.policy b/seccomp_policy/mediacodec.policy
deleted file mode 100644
index 81d042b..0000000
--- a/seccomp_policy/mediacodec.policy
+++ /dev/null
@@ -1,12 +0,0 @@
-# device specific syscalls
-# extension of services/mediacodec/minijail/seccomp_policy/mediacodec-seccomp-arm.policy
-pselect6: 1
-eventfd2: 1
-sendto: 1
-recvfrom: 1
-_llseek: 1
-sysinfo: 1
-getcwd: 1
-getdents64: 1
-inotify_init1: 1
-inotify_add_watch: 1
-- 
2.7.4

